Microsoft.CodeAnalysis.CSharp.CommandLine.UnitTests.CommandLineTests.IOFailure_OpenOutputFile fails in presence of localized resources

Assemblies Run
F:\Roslyn\Main\Open\Binaries\Debug\Roslyn.Compilers.CSharp.CommandLine.UnitTests.dll
Summary
Tests run: 179   Failures: 1, Skipped: 0, Run time: 34.729s

Failed tests
0.016s✘ Microsoft.CodeAnalysis.CSharp.CommandLine.UnitTests.CommandLineTests.IOFailure_OpenOutputFile : Assert.Contains() failure: Not found: error CS2012: Cannot open 'C:\Users\alekseyt\AppData\Local\Temp\RoslynTests\test.exe' for writing In value: !FLxft!error Ё! CS2012: !IDo7Z!Cannot open 'C:\Users\alekseyt\AppData\Local\Temp\RoslynTests\test.exe' for writing -- 'Ошибка ввода-вывода.' ЁЧЫюр! 

   в Xunit.Assert.Contains(String expectedSubstring, String actualString)
   в Microsoft.CodeAnalysis.CSharp.CommandLine.UnitTests.CommandLineTests.IOFailure_OpenOutputFile() в C:\Builds\Roslyn\Main-PerCheckin-Debug\src\Open\src\Compilers\CSharp\Test\CommandLine\CommandLineTests.cs:строка 5965

I believe the test started failing after @jaredpar added/modified it. We should consider adjusting the test to force output in English language by using /preferreduilang switch as other tests do.

See #821 for repro steps.

--- a/src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs
+++ b/src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs
@@ -6167,7 +6167,7 @@ class Program2
         {
             string sourcePath = MakeTrivialExe();
             string exePath = Path.Combine(Path.GetDirectoryName(sourcePath), "test.exe");
-            var csc = new MockCSharpCompiler(null, _baseDirectory, new[] { "/nologo", $"/out:{exePath}", sourcePath });
+            var csc = new MockCSharpCompiler(null, _baseDirectory, new[] { "/nologo", "/preferreduilang:en", $"/out:{exePath}", sourcePath });
             csc.FileOpen = (file, mode, access, share) =>
             {
                 if (file == exePath)

